model/cinemaSessions.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+var CinemaSessionSchema = new Schema({
```

Comment:
What is the difference between your schema and Vova's?

Reply:
There's no difference, I've used new because I didn't have old one